src/main/java/org/folio/des/converter/aqcuisition/EdifactOrdersExportConfigToTaskTriggerConverter.java

```diff
@@ -38,17 +32,20 @@ public List<ExportTaskTrigger> convert(ExportConfig exportConfig) {
     Optional.ofNullable(specificParameters.getVendorEdiOrdersExportConfig())
       .map(VendorEdiOrdersExportConfig::getEdiSchedule)
       .ifPresent(ediSchedule -> {
-        ScheduleParameters scheduleParameters = ediSchedule.getScheduleParameters();
-        if (scheduleParameters == null || !ediSchedule.getEnableScheduledExport() || NONE.equals(scheduleParameters.getSchedulePeriod())) {
-          return;
-        }
-        if (scheduleParameters.getId() == null) {
-           scheduleParameters.setId(UUID.fromString(exportConfig.getId()));
+        ScheduleParameters scheduleParams = ediSchedule.getScheduleParameters();
+        if (isScheduleTimePresent(scheduleParams)) {
+          if (scheduleParams.getId() == null) {
+            scheduleParams.setId(UUID.fromString(exportConfig.getId()));
+          }
+          scheduleParams.setTimeZone(scheduleParams.getTimeZone());
+          var trigger = new AcqBaseExportTaskTrigger(scheduleParams, null, ediSchedule.getEnableScheduledExport());
+          exportTaskTriggers.add(trigger);
         }
-        scheduleParameters.setTimeZone(scheduleParameters.getTimeZone());
-        var trigger = new AcqBaseExportTaskTrigger(scheduleParameters, null, ediSchedule.getEnableScheduledExport());
-        exportTaskTriggers.add(trigger);
     });
     return exportTaskTriggers;
   }
+
+  private boolean isScheduleTimePresent(ScheduleParameters params) {
+    return !Objects.isNull(params) && !Objects.isNull(params.getScheduleTime());
```

Comment:
You can replace isNull method to nonNull

Reply:
Changed, thanks